### PR TITLE
feat: add diamonds webgl point series example

### PR DESCRIPTION
**Pull request by OliverForeman**
Adds the diamonds example rendering around 54k points

### CODE REVIEW

**Review by adilparvez — APPROVED** [review]

**Review by chrisprice — CHANGES_REQUESTED** [review]
Needs to be brought inline with the new examples format

File: examples/diamonds/index.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+const width = 920;
```
Comment (chrisprice) [inline examples/diamonds/index.js:1]: Should we be defining a fixed width/height?

File: examples/diamonds/index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<html>
```
Comment (chrisprice) [inline examples/diamonds/index.html:1]: Match the html structure of the other charts

File: examples/diamonds/index.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+const pointSeries = fc
+    .seriesWebglPoint()
+    .crossValue(d => d.carat)
+    .mainValue(d => d.price)
+    .size(4)
+    .defined(() => true)
+    .equals(d => d.length)
+    .decorate(program => {
+        fc.pointFill().color([1, 0, 0, 1])(program);
```
Comment (chrisprice) [inline examples/diamonds/index.js:12]: These need updating to match the new API

File: examples/diamonds/index.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+        ])
+        .scaleExtent([1, 10])
+        .translateExtent([
+            [0, 0],
+            [width, height]
+        ])
+        .on('zoom', () => {
+            xScale.domain(d3.event.transform.rescaleX(xScaleCopy).domain());
+            yScale.domain(d3.event.transform.rescaleY(yScaleCopy).domain());
+            d3.select('d3fc-group')
```
Comment (chrisprice) [inline examples/diamonds/index.js:51]: This should just render the chart

**Review by chrisprice — CHANGES_REQUESTED** [review]

**Pull request merged**